vllm/scalar_type.py

```diff
@@ -295,6 +299,12 @@ def float_(cls, exponent: int, mantissa: int, finite_values_only: bool,
         ret.id  # noqa B018: make sure the id is cached
         return ret
 
+    @classmethod
+    def from_id(cls, scalar_type_id: int):
+        if scalar_type_id not in _SCALAR_TYPES_ID_MAP:
+            raise ValueError("")
+        return _SCALAR_TYPES_ID_MAP[scalar_type_id]
```

Comment:
FYI @LucasWilkinson

Reply:
some comments:

1. The error msg for `ValueError` is missing, I have added it just now.
2. Torch ops do not support using `ScalarType` as an argument type, but I need to pass quant_type to `torch.ops.vllm.fused_marlin_moe`. Therefore, I can only pass quant_type_id and convert it to quant_type internally. So I need this.

Reply:
These comments would be useful to put in the function description, if other users run into the same issues